src/libcrun/error.c

```diff
@@ -70,8 +72,14 @@ crun_error_wrap (libcrun_error_t *err, const char *fmt, ...)
   char *swap;
   int ret;
 
-  if (err == NULL || *err == NULL)
-    return 0;
+  assert (err != NULL);
```

Comment:
for the rest of the code, we used something like `return crun_make_error (err, 0, "internal error");`

because there is the risk that an assert would produce output that is ignored (when setting up the child process)